### PR TITLE
update tencentcloud

### PR DESCRIPTION
**Pull request by hellertang**

### CODE REVIEW

File: providers/tencentcloud/gaap.go [hunk 1]
```diff
@@ -37,15 +39,29 @@ func (g *GaapGenerator) InitResources() error {
 	if err := g.loadProxy(client); err != nil {
 		return err
 	}
-	if err := g.loadRealServer(client); err != nil {
-		return err
-	}
+	/*
+		if err := g.loadRealServer(client); err != nil {
```
Comment (sergeylanzman) [inline providers/tencentcloud/gaap.go:43]: Function  loadRealServer still?

File: providers/tencentcloud/gaap.go [hunk 2]
```diff
@@ -37,15 +39,29 @@ func (g *GaapGenerator) InitResources() error {
...
+			return err
+		}
+		if err := g.loadCertificate(client); err != nil {
```
Comment (sergeylanzman) [inline providers/tencentcloud/gaap.go:46]: same?

File: providers/tencentcloud/subnet.go [hunk 3]
```diff
@@ -37,6 +37,17 @@ func (g *SubnetGenerator) InitResources() error {
 	}
 
 	request := vpc.NewDescribeSubnetsRequest()
+	/*
+		request.Filters = make([]*vpc.Filter, 0, 1)
+		name := "is-default"
```
Comment (sergeylanzman) [inline providers/tencentcloud/subnet.go:42]: if it's not in use please delete

File: providers/tencentcloud/vpc.go [hunk 4]
```diff
@@ -68,6 +82,65 @@ func (g *VpcGenerator) InitResources() error {
...
+	}
+
+	return nil
+}
+
+func (g *VpcGenerator) loadSubnets(client *vpc.Client, vpcId, resourceName string) error {
+	request := vpc.NewDescribeSubnetsRequest()
+	request.Filters = make([]*vpc.Filter, 0, 1)
+	/*
+		defaultName := "is-default"
```
Comment (sergeylanzman) [inline providers/tencentcloud/vpc.go:96]: same

**Review by sergeylanzman — CHANGES_REQUESTED** [review]

File: providers/tencentcloud/gaap.go (hunk 1 above)
Comment (hellertang) [reply providers/tencentcloud/gaap.go]: fix

File: providers/tencentcloud/gaap.go (hunk 2 above)
Comment (hellertang) [reply providers/tencentcloud/gaap.go]: fix

File: providers/tencentcloud/vpc.go (hunk 4 above)
Comment (hellertang) [reply providers/tencentcloud/vpc.go]: fix

File: providers/tencentcloud/subnet.go (hunk 3 above)
Comment (hellertang) [reply providers/tencentcloud/subnet.go]: fix

**Review by sergeylanzman — APPROVED** [review]

**Pull request merged**